src/app/_components/common/Spiner.tsx

```diff
@@ -0,0 +1,5 @@
+export function Spinner() {
```

Comment:
We generally use arrow functions to define components and functions across all the project. I see that you're defining some components with regular function definitions and another ones with arrow function. You should pick one method and be consistent with it.